hack/mock-unittest-data/yarn.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+set -o errexit -o nounset -o pipefail
+
+cat << banner-end
+--------------------------------------------------------------------------------
+Generating mock data for yarn unit tests
+--------------------------------------------------------------------------------
+banner-end
+
+tmpdir=$(dirname "$(mktemp --dry-run)")
+
+git clone https://github.com/cachito-testing/cachi2-yarn-berry \
+    --depth=1 \
+    --single-branch \
```

Comment:
`--depth` implies `--single-branch` so technically you don't need it. However, explicit is usually better than implicit, so let's keep it this way.